boards/arduino-atmega-common/include/arduino_pinmap.h

```diff
@@ -156,6 +156,28 @@ extern "C" {
 #define ARDUINO_PIN_A15         ARDUINO_PIN_69
 #endif
 
+/* arduino ADC notation compatibility */
+#define ADC0                     0
+#define ADC1                     1
+#define ADC2                     2
+#define ADC3                     3
+#define ADC4                     4
+#define ADC5                     5
+#if defined (CPU_ATMEGA2560) || defined (CPU_ATMEGA1281)
+#define ADC6                     6
+#define ADC7                     7
+#endif
+#ifdef CPU_ATMEGA2560
+#define ADC8                     8
+#define ADC9                     9
+#define ADC10                    10
+#define ADC11                    11
+#define ADC12                    12
+#define ADC13                    13
+#define ADC14                    14
+#define ADC15                    15
```

Comment:
I will remove the ifdef jungle, by removing the superfluous MCU. So I think, that the ADC defines can be remain in "boards/arduino-atmega-common/in clude/arduino_pinmap", because it includes already pin mapping of these MCU's.